packages/contentstack-bootstrap/src/bootstrap/utils.ts

```diff
@@ -189,6 +189,24 @@ const envFileHandler = async (
       }\nCONTENTSTACK_LIVE_PREVIEW=${livePreviewEnabled}\nCONTENTSTACK_LIVE_EDIT_TAGS=false`;
       result = await writeEnvFile(content, filePath);
       break;
+    case 'compass-app':
+      fileName = '.env';
+      filePath = pathValidator(path.join(clonedDirectory, fileName));
```

Comment:
@sunil-lakshman , could you please verify with @cs-raj  if it is preferable to use his utility function rather than pathValidator since he has developed one to address SRE issues?
If possible try to update everywhere in **bootstrap/utils.ts**